### PR TITLE
Update azure-pipelines.yml

### PR DESCRIPTION
**Pull request by ar13pit**
Merge after tue-robotics/tue-env#470

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Good CI: https://dev.azure.com/tue-robotics/tue-robotics/_build/results?buildId=1786&view=logs&j=8fa98ab2-9222-5bf5-d28a-b1964f727242

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-RoboCup challenge implementations
-This is to high-level code that TU Eindhoven's AMIGO robot uses to perform RoboCup@Home challenges.
+[![Build Status](https://dev.azure.com/tue-robotics/tue-robotics/_apis/build/status/tue-robotics.tue_robocup?branchName=master)](https://dev.azure.com/tue-robotics/tue-robotics/_build/latest?definitionId=1&branchName=master)\
+RoboCup@Home challenge implementations
+This repository contains the high-level code that TU Eindhoven's AMIGO, HERO and SERGIO robots use to perform RoboCup@Home challenges.
 
-Most, if not all, challenges are implemented as (large) hierarchical state machines, using the robot_smach_states library of states/state machines.
-These states use the robot_skills robot abstraction layer.
+Most, if not all, challenges are implemented as (large) hierarchical state machines, using the `robot_smach_states` library of states/state machines.
+These states use the `robot_skills` robot abstraction layer.
```
Comment (ar13pit) [inline README.md]: Isn't this already in master?

Comment (MatthijsBurgh) [reply README.md]: Yep, github is confused

Comment (MatthijsBurgh) [reply README.md]: Just merge master into fix/ci, now GitHub isn't confused anymore

**Pull request merged**